aws/resource_aws_emr_cluster.go

```diff
@@ -879,8 +879,11 @@ func resourceAwsEMRClusterCreate(d *schema.ResourceData, meta interface{}) error
 		}
 		return nil
 	})
+	if isResourceTimeoutError(err) {
+		resp, err = conn.RunJobFlow(params)
+	}
 	if err != nil {
-		return err
+		return fmt.Errorf("Error with instance profile: %s", err)
```

Comment:
🤔 Is this the error message you intended here?

```suggestion
		return fmt.Errorf("error running EMR Job Flow: %s", err)
```